packages/react-components/src/templates/__tests__/TeamProfileOutputs.test.tsx

```diff
@@ -34,8 +33,8 @@ it('renders output cards', () => {
   expect(links).toHaveLength(2);
   const [titleLink, teamLink] = links;
 
-  expect(titleLink.textContent).toMatchInlineSnapshot(`"Title"`);
-  expect(teamLink.textContent).toMatchInlineSnapshot(`"Team Unknown"`);
+  expect(titleLink).toHaveTextContent('Title');
+  expect(teamLink).toHaveTextContent(/Unknown/);
```

Comment:
not case insensitive

Reply:
Yeah, this string comes from the input data not some copy